preview/index.html

```diff
@@ -6,6 +6,11 @@
   </head>
   <body>
     <script>
+      // IMPORTANT: the full functionality of this script for pull requests only takes effect once merged
+      // the file is places in the root directory where the preview app resides, not the version located where the app is served from the url containing the PR number.
+      // it is suspected BUT NOT VERIFIED that a script will place this file at the root of the preview deploy upon merge with develop
```

Comment:
@saanobhaai can you verify how the redirect magic works for PR deployments? It seemed to always be calling on an old version of this file and any changes I made not knowing this didnt take effect.